pipeline_steps/common.groovy

```diff
@@ -3,7 +3,8 @@ import groovy.json.JsonOutput
 
 // Install ansible on a jenkins slave
 def install_ansible(){
-  sh """#!/bin/bash
+  sh """#!/bin/bash -xe
+    cd ${env.WORKSPACE}
```

Comment:
I removed venv as an argument in various places because there is only one venv used. I didn't want to introduce another variable to replace it as a well known path is simpler.  There shouldn't be multiple .venv venvs, the should all point to the same one, unless I missed something.